spec/spec.md

```diff
@@ -1462,6 +1498,23 @@ The following JSON Schema Draft 7 definition summarizes the rules above:
 
 </tab-panels>
 
+## Credential Format Designations
+
+Within the _Presentation Exchange_ specification, there are numerous sections 
+where Verifiers and Holders convey what credential variants they support and 
+are submitting. The following are the normalized references used within the 
+specification:
+
+- `vc`, `vp` - these supported formats are defined in the 
+  [W3C Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) specification,
```

Comment:
Could be helpful to add links to the exact definitions that specify `vc` and `vp` mean respectively. Those are new terms so we should define them:
https://www.w3.org/TR/vc-data-model/#dfn-verifiable-credentials
https://www.w3.org/TR/vc-data-model/#dfn-verifiable-presentations

This is important because `vc` and `vp` could also refer to JWT-encoded claim names defined here and pending registration with IANA:
https://www.w3.org/TR/vc-data-model/#iana-considerations

Reply:
I added pointers to the JWT section in the VC spec, which contains details on the composition of each of these